Tools/CISupport/ews-build/events.py

```diff
@@ -25,12 +25,17 @@
 import json
 import os
 import time
+import twisted
 
+from base64 import b64encode
+from buildbot.process.results import SUCCESS, FAILURE, CANCELLED, WARNINGS, SKIPPED, EXCEPTION, RETRY
 from buildbot.util import service
 from buildbot.www.hooks.github import GitHubEventHandler
+from steps import GitHub
 from twisted.internet import defer
 from twisted.internet import reactor
 from twisted.internet.defer import succeed
+from twisted.internet.protocol import Protocol
```

Comment:
is this used anywhere?

Reply:
No, looks to be left over from a previous change.